barretenberg/cpp/src/barretenberg/relations/relation_parameters.hpp

```diff
@@ -45,6 +45,11 @@ template <typename T> struct RelationParameters {
         return RefArray{ eta, eta_two, eta_three, beta, gamma, public_input_delta, lookup_grand_product_delta };
     }
 
```

Comment:
Yes, when you have a function `f(const T& params)` that internally calls `params.get()`, the function `get` needs to be marked `const` so that the compiler knows that `params` isn't mutated by it inside of the body of `f`, thereby invalidating the claim that the input to `f` is `const`. But we can't have only `const` version because we use the non-const version in a non-constant way already (we have `for (auto& x : params.get())` where we assign to each `x`). Since it's 'just C++' stuff I won't document it.